src/test/java/org/broadinstitute/hellbender/tools/walkers/sv/JointGermlineCNVSegmentationIntegrationTest.java

```diff
@@ -254,18 +253,18 @@ public void testOverlappingEvents(final List<File> inputVcfs) {
         }
 
         //these samples have a variant that doesn't overlap the next call
-        for (final String sample : samplesWithGaps) {
-            sawVariant = false;
-            for (final VariantContext vc : overlappingEvents.getRight()) {
-                if (!sawVariant && !vc.getGenotype(sample).isHomRef()) {
-                    sawVariant = true;
-                    continue;
-                }
-                if (sawVariant) {
-                    Assert.assertTrue(vc.getGenotype(sample).isHomRef()
-                            && (Integer.parseInt(vc.getGenotype(sample).getExtendedAttribute(GATKSVVCFConstants.COPY_NUMBER_FORMAT).toString()) == 2));
-                    break;
-                }
+        sawVariant = false;
+        for (final VariantContext vc : overlappingEvents.getRight()) {
+            final Genotype genotype = vc.getGenotype("NA11829");
+            if (!sawVariant && !genotype.isHomRef()) {
+                sawVariant = true;
+                continue;
+            }
+            if (sawVariant && vc.getEnd() == 23236095) {
+                // Smaller variant nested inside larger hom-var DEL: hom-ref genotype but CN is 0 since it overlaps
```

Comment:
Eww is that really what the representation should be?

Reply:
Yes I think that's how gatk-sv does it as well